mdb-realm/__tests__/helpers.spec.ts

```diff
@@ -11,17 +11,28 @@ class helpersTests {
         return {
             apiKey: "set-your-api-key",
             privateApiKey: "set-your-api-key",
-            atlasUrl: "https://cloud-qa.mongodb.com",
             projectId: "set-your-project-id",
-            clusterName: "",
+            clusterName: "Cluster1",
+            atlasUrl: "https://cloud-qa.mongodb.com",
             realmUrl: "https://realm-qa.mongodb.com",
-            useExistingCluster: true,
         };
     }
 
     @test.skip
```

Comment:
Just curious, why are all the tests skipped?

Reply:
Those are integration tests and require actual credentials to run. They're useful to run locally while developing the action, but are fairly useless on CI (e.g. they don't assert anything). Since those actions are indirectly tested by the SDK workflows, we haven't been very diligent about writing proper tests.